iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/IntegrationTest.java

```diff
@@ -62,6 +62,9 @@ protected void finished(Description description)
     @Rule
     public MultiplexingClientTestRule multiplexingClientTestRule = new MultiplexingClientTestRule();
 
+    @Rule
```

Comment:
Added a new rule to help those running E2E tests on a local box when using  a sub that doesn't have error injection enabled.